pymc/sampling/parallel.py

```diff
@@ -271,15 +291,15 @@ def __init__(
         self._readable = True
         self._num_samples = 0
 
-        zarr_chains_send: list[ZarrChain] | bytes | None = None
-        if zarr_chains_pickled is not None:
-            zarr_chains_send = zarr_chains_pickled
-        elif zarr_chains is not None:
+        traces_send: Sequence[IBaseTrace] | bytes | None = None
+        if traces_pickled is not None:
+            traces_send = traces_pickled
+        elif traces is not None:
             if mp_ctx.get_start_method() == "spawn":
```

Comment:
Note this will be the default soon in linux as well (fork is being deprecated). In case we are not automating something we should be automating